chia/full_node/coin_store.py

```diff
@@ -411,6 +411,83 @@ async def get_coin_states_by_ids(
 
         return coins
 
+    async def batch_coin_states_by_puzzle_hashes(
+        self,
+        puzzle_hashes: List[bytes32],
+        *,
+        min_height: uint32 = uint32(0),
+        include_spent: bool = True,
```

Comment:
is this really supposed to default to true?
In the case of subscribing, I don't think we include spent coins in the initial update at least, but I could be wrong.

Reply:
We do:
https://github.com/Chia-Network/chia-blockchain/blob/main/chia/full_node/full_node_api.py#L1508

You need it for syncing historical transactions. That said, this will be specified by the new wallet protocol message anyways using `InitialStateFilters`. All future coin state updates will be received regardless of whether they're spent or not, but for initial state you will have the option to filter out spent coins for performance.